src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,57 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driveService = (DriverService) injector.getInstance(DriverService.class);
+        testDriverService(driveService);
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        testManufacturerService(manufacturerService);
+    }
+
+    private static void printAllDrivers(DriverService driveService) {
```

Comment:
Method ordering.
The method `printAllDrivers() `should followed below the method `testDriverService()` where it called.
The same about `method` `printAllManufacturers`